AmplifyPlugins/Storage/Sources/AWSS3StoragePlugin/Support/Internal/StorageMultipartUploadSession.swift

```diff
@@ -181,6 +181,7 @@ class StorageMultipartUploadSession {
 
     func startUpload() {
         do {
+            transferTask.notify(progress: Progress(totalUnitCount: 0))
```

Comment:
Given that the main `transferTask` lacks a `sessionTask`, this is the only way to set its `status` to `.inProgress`.

As mentioned above, that is necessary for the `pause` method to work